### PR TITLE
Refactor class dialog formatting using dialog inner components

### PR DESCRIPTION
**Pull request by 1999ZiqiZ**

### CODE REVIEW

**Comment by kevinjihwanlee** [comment]
[Refactor class dialog formatting using dialog inner components](https://trello.com/c/BOwAOhc2/190-refactor-class-dialog-formatting-using-dialog-inner-components)

**Review by kevinjihwanlee — COMMENTED** [review]
Looks good to me! Waiting on Joon for final approval.

File: src/components/calendar/ClassModal.js
Comment (Joonpark13) [inline src/components/calendar/ClassModal.js]: After looking at the new layout, let's get rid of the divider.

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**
